static/app/views/issueDetails/groupReplays/useReplaysFromIssue.tsx

```diff
@@ -45,7 +45,7 @@ function useReplayFromIssue({
   }, [api, organization.slug, group.id, group.project.id]);
 
   const eventView = useMemo(() => {
-    if (!replayIds.length) {
+    if (!replayIds) {
```

Comment:
This was an important check.

Before we had no way to differentiate between 'the fetch hasn't happened yet' and 'the fetched returned no replayIds' because we used `[]` for both cases.

Now we have `undefined` for the former, and `[]` for the latter.